### ISSUE TITLE
Bad links (Verify bad link on attachments) - list of events

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#2842]
Bad links (Verify bad link on attachments) - list of events and output the search index filter to easily clean-up the broken events.

**Comment by StefanKelm** [comment]
Sounds cool - that was the question I raised in https://github.com/MISP/MISP/issues/2732#issuecomment-351073680 ;)

**Comment by iglocska** [comment]
Yep indeed. I've been slow as hell with going through issues, you've identified like 70% of the stuff we run into way before we do ;)

**Comment by adulau** [comment]
@StefanKelm Cool. Then we can solve 2 issues at once ;-)

**Comment by saeeda12** [comment]
Is this still an ongoing issue? I'm on v2.4.107 (6f1f428d13927633bc32e18cd1a459c8807a1def) and just got that error as well after deleting an event.
